synapse/handlers/room.py

```diff
@@ -33,6 +33,7 @@
 import attr
 from typing_extensions import TypedDict
 
+import synapse.events.snapshot
```

Comment:
Nit: I think we usually prefer `from synapse.events.snapshot import EventContext`.

Reply:
I would have gone for that... except we define our own `EventContext` in this file:
https://github.com/matrix-org/synapse/blob/051a1c3f220938a0ea1a5b328c268bdb3d1ad592/synapse/handlers/room.py#L95-L103

Reply:
😠 😠 😠 😢 🤯 

You make a valid point.